src/main/scala/codacy/dockerApi/DockerEngine.scala

```diff
@@ -46,11 +46,18 @@ abstract class DockerEngine(Tool: Tool) {
         }))
 
         log("tool started")
-        Tool(
-          path = sourcePath,
-          conf = maybePatterns,
-          files = maybeFiles
-        )
+        try {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/Codacy/codacy-engine-scala-seed/file/2393792812/issues/source?bid=2836226&fileBranchId=3230242#l49)